packages/frontend/src/__tests__/commentsList.js

```diff
@@ -1,37 +1,65 @@
 import { screen, render, waitFor } from '@testing-library/react'
 import CommentsList from '../components/postsList/commentsList'
+import PostContext from '../context/Post'
 
-// mocked props
-const commentIds = ['1', '2', '3']
-const page = '/post'
-const mockLoadMoreComments = jest.fn()
-
-test('should render CommentLists correctly *without* comments', () => {
-    render(
-        <CommentsList
-            commentIds={0}
-            page={page}
-            loadMoreComments={mockLoadMoreComments}
-        />
+const renderCommentsField = (postData, commentIds, page, loadMoreComments) => {
+    return render(
+        <PostContext.Provider value={postData}>
+            <CommentsList
+                commentIds={commentIds}
+                page={page}
+                loadMoreComments={loadMoreComments}
+            />
+        </PostContext.Provider>
     )
-    expect(screen.getByText(/it's empty here./i)).toBeInTheDocument()
-    expect(
-        screen.getByText(/people just being shy, no post yet./i)
-    ).toBeInTheDocument()
-    expect(screen.queryByText(/load more posts/i)).not.toBeInTheDocument()
+}
+
+test('should render CommentLists correctly with data and populated commentIds array', () => {
+    const commentIds = ['1']
+    const page = '/post'
+
+    const postData = {
+        commentsById: {
+            1: {
+                id: '1',
+                content: 'content from commentsById',
+                createdAt: '00',
+                reputation: 30,
+                epoch_key: 'epoch_key test',
+            },
+        },
+    }
+
+    renderCommentsField(postData, commentIds, page, jest.fn())
+    expect(screen.getByText(/post by/i)).toBeInTheDocument()
+    expect(screen.getByText(/epoch_key test/i)).toBeInTheDocument()
```

Comment:
It took me a moment to realize this is the epoch key. Might be good to use a variable instead of hardcoding.

Reply:
Ah, of course. Will implement that.